### DISCUSSION TITLE
Fix http link in readme.md and bump the pub.dev score 5 pts please

### DISCUSSION THREAD

**Discussion by neiljaywarner** [zino-app/graphql-flutter#1019] [Show and tell]
https://github.com/zino-hofmann/graphql-flutter/blob/2e39e66b3f33d035a0a8e86448c930b008a0d0b5/README.md?plain=1#L132
(the PRs badge)
https://makeapullrequest.com/ still works

**Comment by vincenzopalazzo** [comment]
I think this required clarification to me.

What is wrong with the https://makeapullrequest.com/ and what are the pub score that is old in the Readme?

**Comment by neiljaywarner** [comment]
The s is missing.
Basically project looks great except pub.dev score a bit low.

That's a quick way to gain 5 points

....sent from my phone

**Comment by vincenzopalazzo** [comment]
We should be close to the maximum point https://pub.dev/packages/graphql_flutter/versions/5.0.1-beta.5/score, and this PR should improve too https://github.com/zino-hofmann/graphql-flutter/pull/1021
